docs/about/get-involved.md

```diff
@@ -0,0 +1,9 @@
+# **Get Involved**
```

Comment:
I would like we encourage more people to get involved in the community. We can work on this on another PR but here at least we could add links to 
- [Why contribute to open source?](https://opensource.guide/how-to-contribute/#why-contribute-to-open-source)

Reply:
Done 👍